site/en/docs/devtools/accessibility/reference/index.md

```diff
@@ -48,25 +48,26 @@ In general, use the Audits panel to determine if:
 To audit a page:
 
 1.  Go to the URL that you want to audit.
-2.  In DevTools, click the **Audits** tab. DevTools shows you various configuration options.
+2.  In DevTools, click the **Lighthouse** tab (called 'Audits' in earlier Chrome versions). DevTools shows you various configuration options.
```

Comment:
```suggestion
2.  In DevTools, click the **Lighthouse** tab. DevTools shows you various configuration options.
```

As you noted, it changed in Chrome 83. A reference to the (very) old version may just confuse readers.